packages/sdk-react/src/components/payables/utils/getInvoiceOverdueDays.ts

```diff
@@ -0,0 +1,38 @@
+import { components } from '@/api';
+
+export const getInvoiceOverdueDays = (
+  invoice:
+    | components['schemas']['PayableResponseSchema']
+    | components['schemas']['ReceivableResponse']
+): false | number => {
+  if (!invoice.due_date) return false;
+
+  const today = new Date();
+  today.setUTCHours(0, 0, 0, 0); // Normalize today to UTC midnight
+
+  const dueDate = new Date(invoice.due_date);
+  dueDate.setUTCHours(0, 0, 0, 0); // Normalize dueDate to UTC midnight
+
+  if (dueDate.getTime() === today.getTime()) return false; // Not overdue if due today
+
+  const statuses: components['schemas']['PayableStateEnum'][] = [
+    'draft',
+    'new',
+    'waiting_to_be_paid',
+    'approve_in_progress',
+  ];
+
+  //ToDo: introduce type guards that will check based on receivable or payable data, intead of type assertion
```

Comment:
Nitpick, but it would be nice to standardise todo notes as TODO: description of note.